python/xorbits/experimental/tests/test_text_dedup.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022-2023 XProbe Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from ..._mars.dataframe.datasource.dataframe import from_pandas as from_pandas_df
+from ...experimental import dedup
+
+
+def test_dedup_execute(setup):
+    words = list("abcdefghijklmnopqrstuvwxyz")
+    df_raw = pd.DataFrame(
```

Comment:
Better use xorbits.pandas to create a DataFrame.

Reply:
done